src/main/java/org/codehaus/plexus/archiver/AbstractUnArchiver.java

```diff
@@ -125,6 +128,16 @@ public void setOverwrite( final boolean b )
         overwrite = b;
     }
 
+    @Override
+    public FileMapper[] getFileMappers(  ) {
```

Comment:
There is no white space between the brackets when there are no arguments (like in the interface `FileMapper[] getFileMappers();`)

Reply:
Fixed by https://github.com/codehaus-plexus/plexus-archiver/pull/100/commits/deacf5e4cb6e6a6e197b0887be687c99b1f6f96a.